app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :post
+  
+  def comment_on_post
```

Comment:
This is not necessary in the end product, but is necessary as I go through my TDD steps. I know it will be removed once I walk through all the steps, but for now it is a place holder to help me work through the entire FHP TDD lessons.  I am working hard to solidify my understanding of how and why it all works so I am jumping through some hoops that in the long run will be removed and unnecessary for the app but are necessary today for me.